spread.yaml

```diff
@@ -86,6 +86,8 @@ environment:
     SNAPD_WORK_DIR: '$(HOST: echo "${SPREAD_SNAPD_WORK_DIR:-/var/tmp/work-dir}")'
     # Threshold used to determine which stake locks have to be collected during runtime
     SNAPD_STATE_LOCK_TRACE_THRESHOLD_MS: '$(HOST: echo "${SPREAD_SNAPD_STATE_LOCK_TRACE_THRESHOLD_MS:-0}")'
+    # Allow mismatch in snapd/kernel versions
+    SNAPD_ALLOW_SNAPD_KERNEL_MISMATCH: '$(HOST: echo "${SNAPD_ALLOW_SNAPD_KERNEL_MISMATCH:-true}")'
```

Comment:
it'd be nice to figure out which specific tests need this, but it can be a followup PR

Reply:
yes - I was thinking this too, but I wanted to be very course initially and once we add a spread test for this then this can be narrowed to the tests that need it (probably all the nested secureboot ones until the pc-kernel in xx/stable has 2.68+).